app/build.gradle

```diff
@@ -40,6 +40,9 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.4.2'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation('io.coil-kt:coil:2.1.0')
+    implementation('org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.3')
```

Comment:
I didn't see where we use this dependency. Can we remove it?

Reply:
@Wadeewee it's used in `NfcScanningActivity::78`, its the dependency for Coroutines 😅

Reply:
Altho removing it still works, I guess the `lifecycle-runtime` dependency includes it 🤔  I'll remove it 😄